src/kernels/helpers.ts

```diff
@@ -269,22 +333,22 @@ export function compareKernels(
         return 1;
     }
 
+    const aLang = a.kernelSpec.language?.toLowerCase();
```

Comment:
Ran into an existing issue here cleaning up tests. We were previously checking case insensitive and C# wasn't matching c#.

Reply:
Is possibleNbMetadatLanguage always lower case?

Reply:
Yes, but now that I think about it, I guarantee that at a higher level. I'll make a change to add that toLower also in this function. Makes more sense to contain that check inside the function that matters instead of assuming a precondition.